src/lib/crypto/kimchi_backend/gadgets/ec_group.ml

```diff
@@ -3922,11 +3927,11 @@ let%test_unit "Ec_group.scalar_mul_full" =
       try Kimchi_pasta.Vesta_based_plonk.Keypair.set_urs_info [] with _ -> ()
     in
 
-    (* Test elliptic curve scalar multiplication with scalar (fully constrained)
-     *   Rows without external checks:  9,239
-     *   Rows with external checks:    51,284
+    (* Test elliptic curve scalar multiplication with scalar (fully constrained, for 256 bit length scalars)
+     *   Rows without external checks:  9239
+     *   Rows with external checks:     >2^16 
```

Comment:
Oh well, my understanding is that we have around 2^16 rows. In any case, those changes to the tests correspond to a previous already approved and merged PR [here](https://github.com/MinaProtocol/mina/pull/13485)

Basically, with more range checks, now the circuit those tests were creating became much larger and thus not longer fit. But it is not really worth expanding on that at this stage, since that will change again as soon as we modify the rest of things related to the fix.